src/pages/blog/20230127-january-update.md

```diff
@@ -0,0 +1,101 @@
+
+
+## Founding the Mage-OS Association
+
+In full transparency, we are still waiting for the Mage-OS Association foundation to be finalized. Our first application and appeal were returned by the court of Warsaw for statutory changes. We’ve been working with our lawyers and the court to resolve the situation. The latest word is the court and our lawyers are treating our case as a top priority, and we hope to have good news soon.
+
+This delay has prevented us from opening a bank account or conducting any business that requires a registered entity. Until then, we aren’t able to open our membership program, or collect or disburse funds to speed up the pace of our activities.
+
+The silver lining is that this has given us more time to prepare the membership program, budget, and plans for the coming year. Here’s a sneak preview of the Mage-OS supporter program badges:
+
+![A preview of the Mage-OS Association's upcoming membership badges, courtesy of Amasty](/blog/2023/2023-01-27-mageos-badge-preview.png)
+
+A massive thank you to Sergej Derzap and the Amasty graphic team for their work designing these badges!
+
+
+## Technical Update
+
+We have discovered that the nightly-mirror builds are an excellent way to be aware of what is happening inside Magento Open Source.
+
+Recently, an issue in the Magento Open Source repository caused the Mage-OS nightly builds to fail. Adobe implemented a workaround in the 2.4.6 beta builds (outside of the Magento repository), which we have also applied to the nightly builds until a proper and permanent solution is applied.
+
+In brief: A new [Magento_Elasticsearch8](https://github.com/mage-os/mirror-magento2/tree/2.4-develop/app/code/Magento/Elasticsearch8) module is added to Magento Open Source 2.4.6, and requires `elasticsearch/elasticsearch` library version 8. That directly conflicts with the existing `Magento_Elasticsearch7` module, which requires version 7 of the same library. Only one can be installed at a time, so this causes an unresolvable dependency failure and a broken build.
+
+In 2.4.6-beta1, Adobe resolved this by removing the requirement from `Magento_Elasticsearch8`. However, that makes the `Elasticsearch8` module non-functional, because composer will always install `elasticsearch/elasticsearch` version 7. Damien Retzinger opened [an issue](https://github.com/magento/magento2/issues/36687) on the Magento Open Source repository to track and discuss this problem.
+
+These kinds of issues can go undetected for a long time between releases, and this is another reason we are happy to have the nightly builds in place as an early warning system.
+
+During the holiday season most of us involved in the technical development of Mage-OS took a break, though some continued to power on (shout out to Adam Crossland!). Since the new year, momentum has picked up again. Work is progressing on the continuous integration pipeline, which is critical to us accepting contributions to the Mage-OS Distribution.
```

Comment:
Just because of my fussiness: I think that the dot in `Crossland!).` is redundant because there is already the exclamation mark that ends the sentence. At least in Italian, we would not use the dot, I don't know if English follows the same rule.

Reply:
It's (more or less) proper English, but thank you for the feedback!